events/sr2017/kickstart.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: news
+title: SR2017 Kickstart
+permalink: /events/sr2017/kickstart
+date: 2016-10-03
+---
+
+The venues for Student Robotics 2017 kickstart are:
+
+- University of Southampton
+- University of Bristol
+- The Kings Centre, Oxford
+
+Teams are welcome to attend any venue they wish. However; they should inform their local team co-ordinators which venue they will attend, as we will need to arrange for the robotics kits to be at the venue they choose, and some venues have vehicle access restrictions which will need to be arranged beforehand.
+
+Below is an incomplete table of venue-dependent information, which will be completed very soon.
+
+| Venue                     | How to get there                                     | Requires a laptop? | Nearby Food Locations | Finish Time |
+|---------------------------|------------------------------------------------------|--------------------|-----------------------|-------------|
+| University of Southampton | [Getting to campus][soton-campus-directions]. on-campus directions **coming soon** | No                 | [Map][soton-food-map] | 17:00 |
+| University of Bristol     | [Getting to campus][bris-campus-directions]. on-campus directions **coming soon** | Yes                | Coming soon  | 16:00 |
+| The Kings Centre, Oxford  | [Download directions][oxford-directions]             | Yes                | Students should bring their own food | 16:00 |
```

Comment:
We don't need to host this PDF, especially as we likely don't have a license to do do.  Link to it here instead: https://www.kingscentre.co.uk/perch/resources/directions-to-tkc-2015.pdf